src/main/java/teammates/common/util/Const.java

```diff
@@ -908,10 +905,8 @@ private Const() {
         public static final String SEARCH_STUDENTS = "searchstudents";
         public static final String SEARCH_FEEDBACK_SESSION_DATA = "searchfeedbacksessiondata";
 
-        public static final String ERROR_FEEDBACK_EMAIL_RECEIVER_ADDRESS = "errorfeedbackemailreceiveraddress";
         public static final String ERROR_FEEDBACK_EMAIL_SUBJECT = "errorfeedbackemailsubject";
-        public static final String ERROR_FEEDBACK_EMAIL_CONTENT = "errorfeedbackemailcontent";
-        public static final String ERROR_FEEDBACK_URL_REQUESTED = "errorfeedbackrequestedurl";
+        public static final String ERROR_FEEDBACK_REQUEST_ID = "errorfeedbackrequestid";
```

Comment:
Then it might as well be `REQUEST_ID` as it is not necessarily "error"? The usage of this parameter is only for error reports, so I think the name is appropriate.